### PR TITLE
Fix specs for multitenancy

### PR DESCRIPTION
**Pull request by fastjames**
Fix specs within this gem after our recent modifications for engine compatibility. This is mainly to make the extension compatible with recent changes for multitenancy.

### CODE REVIEW

**Comment by fastjames** [comment]
[ch6238](https://app.clubhouse.io/enginecommerce/story/6238/make-spree-user-multitenant-safe-throughout-engine-platform)

File: app/controllers/spree/api/gift_cards_controller.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 module Spree
   module Api
     class GiftCardsController < Spree::Api::BaseController
-      skip_before_action :load_user, only: [:show, :update, :redeem]
-      skip_before_action :authenticate_user, only: [:show, :update, :redeem]
+      # skip_before_action :load_user, only: [:show, :update, :redeem]
```
Comment (nspinazz89) [inline app/controllers/spree/api/gift_cards_controller.rb]: why comment these out instead of just removing them?

**Review by nspinazz89 — APPROVED** [review]
looks good :cake:

**Review by cnorm35 — APPROVED** [review]

File: app/controllers/spree/api/gift_cards_controller.rb (hunk 1 above)
Comment (fastjames) [reply app/controllers/spree/api/gift_cards_controller.rb]: Good catch, fixed.

**Pull request merged**